### DISCUSSION TITLE
I have list of canvas in which i am using react virtual, When i  zoom in the canvas i am resizing the virtual item but when i zoom to certain number i get scrolled to the last virtual item

### DISCUSSION THREAD

**Discussion by amirthapamgr** [TanStack/virtual#791] [Q&A]
I have list of canvas in which i am using react virtual, When i  zoom in the canvas i am resizing the virtual item but when i zoom to certain number i get scrolled to the last virtual item

**Comment by piecyk** [comment]
It's because of our auto adjustment logic, you can control it via `shouldAdjustScrollPositionOnItemSizeChange` it's an instance method, so to turn it of `instance.shouldAdjustScrollPositionOnItemSizeChange = () => false` after zoom just reset it to `shouldAdjustScrollPositionOnItemSizeChange = undefined`
